src/state/slices/stakingDataSlice/selectors.ts

```diff
@@ -72,6 +73,42 @@ export const selectTotalStakingDelegationCryptoByAccountSpecifier = createSelect
   }
 )
 
+export const selectTotalStakingDelegationCrypto = createSelector(
+  selectStakingData,
+  // We make the assumption that all delegation rewards come from a single denom (asset)
```

Comment:
we can do better than this - return an object keyed by the asset id with the crypto balance as the value, like all the other selectors do, then we don't have to change in future, and below, we can get the market data for that asset and not hardcode cosmos